SPECS/kernel/config

```diff
@@ -293,11 +293,8 @@ CONFIG_TRACEPOINTS=y
 #
 CONFIG_CRASH_CORE=y
 CONFIG_KEXEC_CORE=y
-CONFIG_HAVE_IMA_KEXEC=y
-CONFIG_KEXEC=y
-CONFIG_KEXEC_FILE=y
-# CONFIG_KEXEC_SIG is not set
-# CONFIG_KEXEC_JUMP is not set
+# CONFIG_KEXEC is not set
+# CONFIG_KEXEC_FILE is not set
```

Comment:
@allenpais `CONFIG_KEXEC_FILE` still needs to be enabled. We just want to disable `CONFIG_KEXEC` because it doesn't support signature verification.